src/shared/components/checkbox/Checkbox.scss

```diff
@@ -1,53 +1,84 @@
 @import 'src/styles/common';
 
-.checkboxHolder {
-  display: grid;
-  grid-template-columns: 27px 1fr;
+.wrapper {
+  display: inline-block;
 }
 
-.defaultCheckbox {
+.grid {
+  display: inline-grid;
+  grid-template-columns: auto 1fr;
+  column-gap: var(--checkbox-distance-to-label, 12px);
```

Comment:
Why are we defining these var?

Reply:
If we ever need to tweak the distance between the checkbox and the label. I thought this is within the possibilities of what we may be asked to do.

Reply:
Removed the var 🤷‍♂️ 

_(Also, the distance between the checkbox and the label is now defined via label padding)_